composer.json

```diff
@@ -24,6 +24,7 @@
         "composer-plugin-api": "^2.0",
         "composer-runtime-api": "^2.0",
         "acquia/drupal-environment-detector": "^1.5.3",
+        "acquia/drupal-recommended-settings": "dev-ACMS-3508",
```

Comment:
This needs to be a stable tag before we can merge

Reply:
@danepowell yes we will update this before our release for DRS.

Reply:
@danepowell  - this is done.